ci/templates/cf-for-k8s-contributions.yml

```diff
@@ -662,6 +663,194 @@ jobs:
         params:
           PGPASSWORD: ((ci_k8s_aws_rds_database_password))
 
+- name: validate-pr-on-gke-external-blobstore
```

Comment:
We just updated the corresponding `validate-pr-on-gke-external-db` job in the `cf-for-k8s-contributions` pipeline to use pooled clusters instead of terraforming them directly to match the version in the `cf-for-k8s-main` pipeline to allow it to run in parallel. Would you mind making the same change here?